src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/ItemStackAndInteger.java

```diff
@@ -20,12 +20,13 @@ public int getInt() {
     }
 
     public ItemStack getItem() {
+        initializeItem();
         return item;
     }
 
     public ItemStackWrapper getItemStackWrapper() {
```

Comment:
Both of these methods should probably be marked as `@Nullable`

Reply:
They weren't originally. Additionally apparently `getItemStackWrapper` is actually nullable if `item` is null... Although it shouldn't so :P

Reply:
Actually put it this way. If the `ItemStack` is null then the wrapper will be null so no they aren't both nullable.